mods/uifile-override.wh.cpp

```diff
@@ -0,0 +1,202 @@
+// ==WindhawkMod==
+// @id              uifile-override
+// @name            UIFILE Override
+// @description     Override DirectUI XML files
+// @version         1.0
+// @author          xalejandro
+// @github          https://github.com/tetawaves
+// @include         *
+// @compilerOptions -luxtheme -lshlwapi
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# UIFILE Override
+Mod that overrides DirectUI XML files same way as OldNewExplorer, loading them
```

Comment:
Can this mod be useful for somebody who's not familiar with OldNewExplorer? I suggest to improve the description and make it clear what the mod does even for those who aren't familiar with OldNewExplorer. Adding screenshots, if relevant, can help.